src/satosa/frontends/oidcop/exceptions.py

```diff
@@ -0,0 +1,2 @@
+class InconsinstentSessionDump(Exception):
```

Comment:
to be removed

Reply:
https://github.com/IdentityPython/SATOSA/pull/378/commits/7faf83572452328df58a503eb61b6b85f89a0106